notesapi/v1/models.py

```diff
@@ -11,12 +12,16 @@ class Note(models.Model):
     course_id = models.CharField(max_length=255, db_index=True)
     usage_id = models.CharField(max_length=255, help_text="ID of XBlock where the text comes from")
     quote = models.TextField(default="")
+    parent = models.ForeignKey('Note', blank=True, null=True, help_text="Parent note, if this is a comment")
```

Comment:
Thank you. Sorry for the confusion. I'll figure out the proper odl-etiquette for these sorts of things soon, I promise. Just don't want you to waste your time on code that may not live to the end of the week. I'm sure you'd rather be hacking on Go or similar. :smile_cat: